contrib/omhttp/omhttp.c

```diff
@@ -1260,12 +1259,6 @@ serializeBatchLokiRest(wrkrInstanceData_t *pWrkrData, char **batchBuf)
 	}
 
 	for (size_t i = 0; i < numMessages; i++) {
-		valueObj = fjson_object_new_object();
```

Comment:
Do I correctly understand that this code was simply useless?

Reply:
> Do I correctly understand that this code was simply useless?

Yes, valueObj  was not used at all - and I am pretty sure just a leftover from copy&paste of serializeBatchKafkaRest function.

Reply:
thx!